### PR TITLE
Adjust GLPK lib locations to enable Windows builds

### LINKED ISSUE #11: support for windows
Issue: support for windows

There seems to be no conda package for windows. Is there a reason for that? There exist [pypi packages for windows](https://pypi.org/project/swiglpk/).

### PR DESCRIPTION
**Pull request by cdiener**
This is another attempt at getting windows builds working.

Checklist
* [X] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [X] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed) - does not apply
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Fixes #11.

Pretty similar to #12 but copies the GLPK header and library where swiglpk expects them to be. Using GCC should not be necessary as this uses the normal build system and will work with MSVC.

### CODE REVIEW

**Comment by cdiener** [comment]
Is there a way to test the windows build? looks like the CI only runs Linux and Mac...

**Comment by isuruf** [comment]
@conda-forge-admin, please rerender

**Comment by cdiener** [comment]
@druvus It would be great if you could take a look. I think everything is working now.

**Comment by cdiener** [comment]
@conda-forge/core

File: recipe/meta.yaml [hunk 1]
```diff
@@ -34,15 +30,17 @@ test:
...
-    if you are interested in a python-based mathematical programming language). 
+    swiglpk is not a high-level wrapper for GLPK (take a look at optlang
+    if you are interested in a python-based mathematical programming language).
     It just provides plain vanilla swig bindings to the underlying C library.
   dev_url: https://github.com/biosustain/swiglpk
 
 extra:
   recipe-maintainers:
     - druvus
+    - sthiele
```
Comment (isuruf) [inline recipe/meta.yaml]: @sthiele, can you confirm you want to be a maintainer?

Comment (sthiele) [reply recipe/meta.yaml]: Thanks, but I have to pass.

Comment (cdiener) [reply recipe/meta.yaml]: Sad to hear that, I'll remove you.

**Comment by cdiener** [comment]
@conda-forge/core I adjusted the maintainer list, thanks!

**Pull request merged**